src/components/date/__spec__.js

```diff
@@ -640,4 +641,24 @@ describe('Date', () => {
       ]);
     });
   });
+
+  describe('DayPickerStyle', () => {
+    const shallowRender = (props, renderer = shallow) => {
+      return renderer(
```

Comment:
renderer could be changed to `TestRenderer.create` as in this test you're not using other renderes

Reply:
Fixed, thanks